### PR TITLE
P1-2: -Todo list was created

### PR DESCRIPTION
**Pull request by NikitaHlinsky-0308**

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+.checkbox:checked + .checkmark {
+    border-color: #019b01;
+}
+
+/* .checkbox:checked > .text {
+    color: crimson;
+    text-decoration: line-through;
+} Dont know how to make text "line-through" when checkbox is selected  */
```
Comment (i-barysiuk) [inline src/css/style.css:107]: try to use + instead of >

**Review by i-barysiuk — CHANGES_REQUESTED** [review]

File: src/css/style.css (hunk 1 above)
Comment (NikitaHlinsky-0308) [reply src/css/style.css:107]: Tried, it still didn't work, but I suggested it's cause span in which I put check mark(".checkmark"), without it code is works.
Screenshot 2020-12-24 at 18 07 40
Screenshot 2020-12-24 at 18 08 42
On second pic both checkboxes is checked.

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+body {
+    background-color: #d1cece;
+    color: #6d6d6d;
+    font-family: 'Raleway', sans-serif;
+}
+
+.title {
+    margin: 0 auto;
+    padding: 2rem 0;
+    width: 30vw;
```
[the remark this answers is not in the document]
Comment (NikitaHlinsky-0308) [reply src/css/style.css]: Fixed by a media query

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <span class="text">Lorem ipsum dolor sit amet. </span>
+        </label>
+        <div class="footer-input">
+            <div class="counter">
+                <p>2 2 left</p>
+            </div>
+            <div class="radio-footer">
+                <div class="form-radio-btn">
+                    <input id="radio-1" type="radio" name="radio" value="1">
+                    <label for="radio-1">btn 1</label>
```
[the remark this answers is not in the document]
Comment (NikitaHlinsky-0308) [reply src/index.html]: fixed.

File: src/css/style.css (hunk 1 above)
Comment (i-barysiuk) [reply src/css/style.css]: the second one is not checked
set checked attr to second input

**Review by i-barysiuk — APPROVED** [review]

**Pull request merged**